README.md

```diff
@@ -9,6 +9,9 @@ Small binary to scrape and store a small window of metrics from the Metrics Serv
 | db-file  | What file to use as a SQLite3 database.  |  `/tmp/metrics.db` |
 | metric-resolution | The resolution at which dashboard-metrics-scraper will poll metrics.  | `1m` |
 | metric-duration | The duration after which metrics are purged from the database. | `15m` |
+| log-level | The log level. | `info` |
+| logtostderr | Log to stderr. | `true` |
+| namespace | The namespace to use for all metric calls. When provided, skip node metrics. | `defaults to cluster level metrics` |
```

Comment:
I'd remove the quotation marks in the defaults column.

Reply:
Do you mean across the column? Or just the ones I added?

Reply:
It's good now.